Letterbook.Adapter.Db/DesignDbOptions.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Letterbook.Adapter.Db;
+
+public class DesignDbOptions : DbOptions
```

Comment:
Yeah, the DbOptions object is read from the config file, in progam.cs
https://github.com/Letterbook/Letterbook/pull/28/files#diff-18ae441c6ad46bdfd4ffba0ec909f140137ca14f4d149bb9e257fad4c82d3cd8R24

Specifically, it's pulled from the `Letterbook.Database` field. This way we're not passing around a single massive options object everywhere, which I think would make horizontal scaling harder in the future. (I have a rant/blog post simmering on how bad we all are as an industry at managing config.)
https://github.com/Letterbook/Letterbook/pull/28/files#diff-dffb3136b367460c07bf2c4e89b15958da6382630d433a5db7bee3f4d7507b31R6-R12

Reply:
And no, it's not an EF thing. I just need a way for the admin to tell the dbadapter what the connection string is, and since we're starting with an ASP.net app, `builder.Services.Configure()` is the obvious route.